fixtures/func-default-values/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+fn simple(amount: i32) -> i32 {
+    amount - 1
+}
+
+fn multiple(left: i32, right: i32) -> i32 {
+    left - right
+}
+
+fn default_last(left: i32, right: i32) -> i32 {
+    left - right
+}
+
+fn default_first(left: i32, right: i32) -> i32 {
+    left - right
+}
+
+fn three_args(a: i32, b: i32, c: i32) -> Vec<i32> {
+    vec![a, b, c]
+}
```

Comment:
Yes yes, I only realized it towards the end that returning a vec of the arguments is so much easier to read. If we land any of this I'll rewrite the other functions as well.